pkg/recommendation/device.go

```diff
@@ -0,0 +1,235 @@
+package recommendation
+
+import (
+	"sort"
+
+	"github.com/pkg/errors"
+	"k8s.io/apimachinery/pkg/api/resource"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+
+	"mayadata.io/cstorpoolauto/types"
+	bdutil "mayadata.io/cstorpoolauto/util/blockdevice"
+)
+
+// nodeCapacityBlockDevice contains key with node name and value with capacityBlockDevice
+type nodeCapacityBlockDevice map[string]capacityBlockDevices
```

Comment:
How are we determining a variable with this name should belong to device.go file?